bundles/rust/rust_mode.moon

```diff
@@ -0,0 +1,29 @@
+--- rust mode for howl from [ta-rust](https://bitbucket.org/a_baez/ta-rust)
+-- See @{README.md} for details on usage.
+-- @author [Alejandro Baez](https://keybase.io/baez)
+-- @copyright 2016
+-- @license MIT (see LICENSE)
+-- @module mode
+
+import style from howl.ui
+
+style.define 'extension', "special"
```

Comment:
Is there really a need for this? I think these might be global definitions, which could interfere with other modes.